### ISSUE TITLE
Function to merge two geojson objects

### ISSUE THREAD

**Issue by Isaac-Leonard** [Turfjs/turf#2243]
I'm currently using version 6.5.0
Would it be possible to add a function to merge to geojson objects into one?
Such as for when collating geojson from different data sources.
So if given two features it would return a feature collection, if given two featureCollections it would combine this into one, etc.
Checking the docs it doesn't seem as though this currently exists although I also could've just missed it.

**Comment by stebogit** [comment]
@Isaac-Leonard I don't think you can merge GeoJSON objects other than `FeatureCollection`s as you suggested (I can't think of an obvious/standard way of merging two polygons or a line and a polygon 🤔 ).
In that case I think you can simply either create a feature collection from a set of features using the `featureCollection` helper (`@turf/helpers`) or merge/add additional features into the `features` property of an existing `FeatureCollection`:

```js
import {point, lineString, featureCollection} from '@turf/helpers';

const pt = point([20, 30]);
const ln = lineString([[-24, 63], [-23, 60], [-25, 65]]);
const poly = polygon([[[-5, 52], [-4, 56], [-2, 51], [-7, 54], [-5, 52]]]);

const collection = featureCollection([pt, ln]);
collection.features = [...set.features, poly];
```

I don't think we need a function for that.
